actions/next-semver/dist/licenses.txt

```diff
@@ -0,0 +1,73 @@
+@actions/core
```

Comment:
Do we need the `actions/next-semver/dist folder` to be checked in?

Reply:
see https://github.com/alphagov/pay-ci/pull/823#discussion_r1108313431